frontend/src/config/shared/translation/english.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
🙌

Reply:
We still need to figure out exactly how to make this work / use it since it won't have all the legends and other things. Maybe we could load them the same way we do it for tests in dev mode